x/auth/vesting/msg_server.go

```diff
@@ -70,8 +74,12 @@ func (s msgServer) CreateVestingAccount(ctx context.Context, msg *types.MsgCreat
 	if msg.Delayed {
 		vestingAccount = types.NewDelayedVestingAccountRaw(baseVestingAccount)
 	} else {
-		sdkctx := sdk.UnwrapSDKContext(ctx)
-		vestingAccount = types.NewContinuousVestingAccountRaw(baseVestingAccount, sdkctx.HeaderInfo().Time.Unix())
+		start := msg.StartTime
```

Comment:
Actually, should we add a check that it does not start in the past?
And that start is a valid time?

Missing the <= 0 check for instance.

Reply:
Whats the effect of starting at the past? That instantly a portion will be vested right?

Reply:
Probably, but that is a weird use case. Avoiding starting in the past can avoid people making fat finger mistakes.